resources/assets/sass/_events.scss

```diff
@@ -323,29 +323,30 @@
     background-color: $brand-white !important;
 }
 
-#devices-section {
-    border: 1px solid #0394a6 !important;
+.ourtabs {
```

Comment:
Is this ourtabs as in, not bootstrap's tabs?

Reply:
Our overrides of bootstrap styling.  Would you like a different name to make that clearer?